shellspy_test.go

```diff
@@ -1,24 +1,57 @@
 package shellspy_test
 
 import (
+	"bytes"
 	"fmt"
+	"os"
 	"os/exec"
 	"shellspy"
+	"strings"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
 )
 
 func TestCommandFromString(t *testing.T) {
 
-	testStrings := []string{"echo", "cat", "ls -la"}
+	input := "echo hello world\n"
+	want := "/bin/echo hello world"
```

Comment:
Looks like we are discarding the err return value.  Does it return anything we can use?  If not, perhaps we don't need it